README.md

```diff
@@ -10,17 +10,31 @@ Test via ```nosetests```.
 ### Usage: 
     POST   on /shopcarts: creates new shopcart.
         Expects JSON body data:
-            item_id: -1
+            item_id: 0
             user_id: int
+            item_name: string, "ring" + str(item_id)
+            quantity: int, range from 0 to 999
+            price: float, range from 0.0 to 0.1
+    GET    on /shopcarts: returns all of the shopcarts.
+    GET    on /shopcarts/<user-id>: returns items in <user-id> shopcart.
+    DELETE on /shopcarts/<user-id>: deletes <user-id> shopcart.
+    POST   on /shopcarts/<user-id>/items: creates new item in <user-id> shopcart.
+        Expects JSON body data:
+            item_id: int
+            quantity: int 
             item_name: string
-            quantity: int
             price: float
-    GET    on /shopcarts: returns list of all shopcarts
-    PUT    on /shopcarts/<user-id>: change item quantities in <user-id> shopcart.
+    GET    on /shopcarts/<user-id>/items/item_id: reads an <item_id> item in a certain <user-id> shopcart
+    PUT    on /shopcarts/<user-id>/items/item_id: updates price and/or quantity of an existing item in shopcart {shopcart_id}
+        Expects JSON body data:
+            quantity: int, positive
+            price: float, non-negative
+    DELETE on /shopcarts/<user-id>/items/item_id: deletes an <item_id> item in a certain <user-id> shopcart
+### To be fixed:
```

Comment:
This route should just be deleted since we are not using PUT that way anymore

Reply:
I or someone else can delete that commented part in routes.py next time

Reply:
Just to confirm, the "PUT" is what we talked in Slack tonight: "PUT    on /shopcarts/<user-id>: change item quantities in <user-id> shopcart. "
If so, I will delete both the part in routes.py and README.md in next commit.

Reply:
Confirmed, but actually I think we should keep it for now. It can be used for if/when we implement the PUT /shopcarts/<shopcart_id> route for batch updating a shopcart, which as the professor said is optional.

Reply:
nvm, Thanks again Timothy~